### PR TITLE
adds some more information and links, fixes "database" instead of "table"

### PR DESCRIPTION
**Pull request by riconeitzel**

### CODE REVIEW

**Comment by riconeitzel** [comment]
I don't know, why the link tests failed … If I click the new link it works just fine. Anyone knows, what's wrong here?

File: guides/plugins/plugins/framework/data-handling/add-complex-data-to-existing-entities.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ class CustomExtension extends EntityExtension
 
 As you can see, we're adding a new `OneToOneAssociationField`. Its parameters are the following, in correct order:
 
-* `propertyName`: The name of the property which should contain the associated entity of type `ExampleExtensionDefinition` in the `ProductDefinition`.
-* `storageName`: Use the `id` column here, which refers to the `id` field of your product. This will be used for the connection to your association.
+* `propertyName`: The name of the property which should contain the associated entity of type `ExampleExtensionDefinition` in the `ProductDefinition`. Property names are usually camelCase (with first character lower cased)
```
Comment (PaddyS) [inline guides/plugins/plugins/framework/data-handling/add-complex-data-to-existing-entities.md:108]:
```suggestion
* `propertyName`: The name of the property which should contain the associated entity of type `ExampleExtensionDefinition` in the `ProductDefinition`. Property names are usually camelCase, with the first character being lower cased.
```

Comment (PaddyS) [reply guides/plugins/plugins/framework/data-handling/add-complex-data-to-existing-entities.md:108]: What do you think? :-)

**Review by PaddyS — CHANGES_REQUESTED** [review]

**Comment by PaddyS** [comment]
@riconeitzel Oof. Thanks once more! Really appreciate your work here. 🎉

**Pull request merged**